boards/arm/nucleo_l552ze_q/nucleo_l552ze_q_ns_defconfig

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32L5X=y
+CONFIG_SOC_STM32L552XX=y
+# 110MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=110000000
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable pinmux
+CONFIG_PINMUX=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+# SYSCLK selection
+CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
+# PLL configuration
+CONFIG_CLOCK_STM32_PLL_SRC_MSI=y
+CONFIG_CLOCK_STM32_MSI_RANGE=6
+#CONFIG_CLOCK_STM32_LSE=y
+#CONFIG_CLOCK_STM32_MSI_PLL_MODE=y
+# produce 110MHz clock at PLL output
+CONFIG_CLOCK_STM32_PLL_M_DIVISOR=1
+CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=55
+CONFIG_CLOCK_STM32_PLL_P_DIVISOR=7
+CONFIG_CLOCK_STM32_PLL_Q_DIVISOR=2
+CONFIG_CLOCK_STM32_PLL_R_DIVISOR=2
+CONFIG_CLOCK_STM32_AHB_PRESCALER=1
+CONFIG_CLOCK_STM32_APB1_PRESCALER=1
+CONFIG_CLOCK_STM32_APB2_PRESCALER=1
+
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Enable MPU
+# Issue: TFM integration sample tfm_ipc doesn't work when the MPU is enabled.
+# If it needs to be enabled, TFM_BOOT_MPU_PROTECTION for the board in TFM needs
+# to be removed.
+#CONFIG_ARM_MPU=y
```

Comment:
Please explain what the problem is with the MPU on the non-secure image. This has not been a problem for any other TrustZone-enabled platforms so far.

Reply:
Zephyr should be the only firmware responsible for configuring the Non-Secure MPU. IF this is configured by parent images, e.g. a bootloader of TF-M, Zephyr is allowed to overwrite it. 
MPU and MemManageFaults in Cortex-M are banked for exactly this reason.

Which sample are you talking about? This is the board defconfig file.

If you comment out MPU here, then no one can use any memory protection features in Zephyr, on this board.

Reply:
I'm talking about TF-M integration sample tfm_ipc.
This an issue specific to L5 TF-M implementation (tracked at https://developer.trustedfirmware.org/T806). I opened the issue #27809 to track it in Zephyr, according to comments made by Erwan.